its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/ScannerMSBuildTest.java

```diff
@@ -199,37 +181,8 @@ void testSampleWithProxyAuth() throws Exception {
     assertThat(seenByProxy).isNotEmpty();
   }
 
-  @Test
-  void testNoProjectNameAndVersion() throws Exception {
```

Comment:
Why isn't this needed anymore?

Reply:
It is exactly the same as `testSample`.

This was here because SQ below 6.6 required project name and version. And this test was testing that it's not needed. Nowadays, it's not needed and we never send it, so it's the same as the first test.